pyvista/examples/planets.py

```diff
@@ -192,22 +190,24 @@ def load_venus(radius=1.0, lat_resolution=50, lon_resolution=100):  # pragma: no
     --------
     >>> import pyvista as pv
     >>> from pyvista import examples
+    >>> mesh = examples.planets.load_venus()
+    >>> texture = examples.planets.download_venus_surface(texture=True)
     >>> pl = pv.Plotter()
     >>> image_path = examples.planets.download_stars_sky_background(
     ...     load=False
     ... )
     >>> pl.add_background_image(image_path)
-    >>> _ = pl.add_mesh(examples.planets.load_venus())
+    >>> _ = pl.add_mesh(mesh, texture=texture)
     >>> pl.show()
 
     """
     sphere = _sphere_with_texture_map(
         radius=radius, lat_resolution=lat_resolution, lon_resolution=lon_resolution
     )
     surface = download_venus_surface(atmosphere=False, texture=True)
-    sphere.textures["surface"] = surface
+    sphere._textures["surface"] = surface
     atmosphere = download_venus_surface(texture=True)
-    sphere.textures["atmosphere"] = atmosphere
+    sphere._textures["atmosphere"] = atmosphere
```

Comment:
Is there a way we could drop a hint about the atmosphere toggle for the texture download function? Maybe not, because we can only use either texture for plotting, and it's not like it was evident that we had two textures here, based on the docstring example alone.

Reply:
I don't want mess with this `planets` module any more than necessary for this deprecation

Reply:
I just meant adding another line of example that shows the other texture being used, but fair enough.